docker/slack-request-verifier/server.py

```
@@ -0,0 +1,189 @@
+        upstream_url = f"{N8N_INTERNAL_BASE_URL}{self.path}"
+        headers = {
+            key: value
+            for key, value in self.headers.items()
+            if key.lower() not in {"host", "content-length", "connection"}
+        }
+        request = urllib.request.Request(
+            upstream_url,
+            data=body,
+            headers=headers,
+            method="POST",
+        )
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)